src/Resources/Resources/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
```

Comment:
remove this empty line too. Only add your change msg line under "Upcoming Release".